Content.Server/Kitchen/EntitySystems/SharpSystem.cs

```diff
@@ -107,14 +108,11 @@ private void OnDoafterComplete(SharpButcherDoafterComplete ev)
         _popupSystem.PopupEntity(Loc.GetString("butcherable-knife-butchered-success", ("target", ev.Entity), ("knife", ev.Sharp)),
             popupEnt, Filter.Entities(ev.User), popupType);
 
-        if (hasBody)
-        {
-            _bodySystem.GibBody(body!.Owner, body: body);
-        }
-        else
-        {
-            QueueDel(ev.Entity);
-        }
+        // since this doubles as harvestable, and we are e.g. harvesting a tree someone has climbed,
+        // we want to pass the destruction event so it can drop them
+        var args = new DestructionEventArgs();
+        RaiseLocalEvent(ev.Entity, args, false);
+        QueueDel(ev.Entity);
```

Comment:
AFAIK destruction events don't cause gibing, so why get rid of that?
Also this should just call `DestructibleSystem.DestroyEntity()` to avoid code duplication and for future proofing if ever destruction raises additional events.